src/main/java/seedu/address/logic/commands/room/SearchRoomCommand.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.logic.commands.room;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_ROOM_NOT_FOUND;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.room.Room;
+
+
+/**
+ * Searches a room according to the given room number.
+ */
+public class SearchRoomCommand extends Command {
+
+    public static final String COMMAND_WORD = "searchroom";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Finds the room with the given room number.\n"
+            + "Parameters: ROOM NUMBER\n"
+            + "Example: " + COMMAND_WORD + " 130";
+
+    public static final String MESSAGE_SUCCESS = "Room %1$s has been found and listed.";
```

Comment:
I think later when you complete the searchroom rmb to remove the "%1$s" as it's a list of string

Reply:
Ah yes, thanks for pointing that out!